tests/integration/sugar/test_transaction.py

```diff
@@ -236,6 +237,26 @@ async def test_payment_high_fee_authorized(self, client):
         self.assertTrue(response.is_successful())
         WALLET.sequence += 1
 
+    @test_async_and_sync(globals())
+    async def test_payment_high_fee_authorized_submit_xalias(self, client):
+        # GIVEN a new Payment transaction
+        response = await submit_transaction_with_alias_async(
```

Comment:
Instead of creating a helper method for this one test, you should instead use `submit_transaction` and add it to the list of dependencies that need to be imported.